site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -12053,6 +12053,9 @@ private void addNewSite(ParameterParser params, SessionState state) {
 					rp.addProperty(TEMPLATE_USED, templateSite.getId());
 				}
 
+				// Enable MathJax, if applies
+				MathJaxEnabler.prepareMathJaxForNewSite(site, state, (List<String>) state.getAttribute(STATE_TOOL_REGISTRATION_SELECTED_LIST));
```

Comment:
Has the user had the opportunity to select the mathjax enabled tools at this point, or is this just going to enable mathjax for all tools?